source/developers-guide/plugin-quick-start/index.md

```diff
@@ -141,7 +141,7 @@ class FrontendSubscriber implements SubscriberInterface
         ];
     }
 
-    public function myDetailCallback(\Enlight_Controller_ActionEventArgs $args)
+    public function myDetailCallback(Enlight_Controller_ActionEventArgs $args)
```

Comment:
I think this is necessary as the Subscriber should have the namespace `namespace SwagSloganOfTheDay\Subscriber`
Like defined in the services.xml